next-env.d.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
+/// <reference path="./.next/types/routes.d.ts" />
```

Comment:
Hm, how was this generated? Do you think this is a relevant change? 

I'm mostly concerned with the "// NOTE: This file should not be edited" below.

Most of the bump was handled by dependabot (sorry,  didn't see your PR; my notifications are LIFO 😅 )